tests/phpunit/api/v3/LoggingTest.php

```diff
@@ -273,6 +273,7 @@ protected function createLegacyStyleContactLogTable() {
   public function testRevert() {
     $contactId = $this->individualCreate();
     $this->callAPISuccess('Setting', 'create', array('logging' => TRUE));
+    sleep(1);
```

Comment:
Can you add a short comment explaing why sleep is required

Reply:
done